src/content/blog/mariadb-dump-breaking-change.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "MariaDB Dump (mysqldump) Breaking Change"
+pubDate: 2024-06-14
+#modifiedDate: 2024-04-23
+summary: What is the breaking change in Mariadb-dump (mysqldump) and what does it mean for my DDEV projects?
+author: Randy Fay
+featureImage:
+  src: /img/blog/2024/06/weeping-sea-lion.png
+  alt: "Dall-E: an image of a seal posed and crying with big tear drops, something like the MariaDB logo"
+  credit: 'Dall-E: an image of a seal posed and crying with big tear drops, something like the MariaDB logo'
+categories:
+  - DevOps
+---
+
+## What happened with MariaDB and MysqlDump?
+
+On May 17, 2024, MariaDB responded to a security issue by creating a new directive in the output of `mariadb-dump`/`mysqldump`. This new directive in the dump file looks like this:
+
+`/*!999999\- enable the sandbox mode */`
+
+This directive/content is incompatible with all versions of the `mysql` or `mariadb` database client from all vendors from before that May 17, 2024. 
```

Comment:
```suggestion
This directive is incompatible with all versions of the `mysql` or `mariadb` database client from all vendors prior May 17, 2024. 
```
not sure if the slash content is necessary? content confused me more than that it explained things. is it content or a directive or both? personally i would stick to just directive?

Reply:
I didn't even know before that `mysqldump` in MariaDB context is just a symlink to `mariadb-dump`, or that MariaDB has been rebranding all of its tools. So that's why I used the slash all the time.

Reply:
10.4 was a symlink to mariadb-dump. 10.5+ was a symlink the other way. As incompatibilities slowly creap in its important for users to know difference.

Reply:
I don't understand your point @grooverdan , but a PR to this would be welcome. The current situation (mariadb 10.11 for example, is that mysqldump is a symlink to mariadb-dump:

```
$ mysql --version
mysql  Ver 15.1 Distrib 10.11.8-MariaDB, for debian-linux-gnu (aarch64) using  EditLine wrapper
rfay@d10-db:~$ ls -l $(which mysqldump)
lrwxrwxrwx 1 root root 12 May 10 23:02 /usr/bin/mysqldump -> mariadb-dump
```